.github/workflows/cypress_tests.yml

```diff
@@ -54,51 +53,52 @@ jobs:
         env:
           PORT: 5000
 
-      - name: Install pnpm
-        uses: pnpm/action-setup@v2
+      - uses: pnpm/action-setup@v2
         with:
           version: latest
-
-      - name: Run and test app
+      - name: Run Webapp
```

Comment:
I wonder here if adding a step to build the webapp, would ensure our product under test more closely resembles what is on production